clickhouse-native-jdbc/src/main/java/com/github/housepower/settings/SettingKey.java

```diff
@@ -830,6 +829,36 @@ public class SettingKey implements Serializable {
             .isSecret()
             .build();
 
+    public static SettingKey ssl = SettingKey.builder()
+            .withName("ssl")
+            .withType(SettingType.Bool)
+            .withDescription("Establish secure connection: True or False")
+            .build();
+
+    public static SettingKey sslMode = SettingKey.builder()
+            .withName("ssl_mode")
```

Comment:
maybe we should follow the name of clickhouse-jdbc.

https://clickhouse.com/docs/en/integrations/java#ssl-properties

Reply:
Do I understand correctly that you want to add validation using a root certificate, as well as a client certificate and key, to this pull request?

Reply:
@sundy-li also I read such comments like: 
1) Client-Side Certificate: If your Java client is running in a controlled environment, using JKS could be more secure and manageable.
2) Server-Side Certificate: If the client certificate is being distributed or is going to be used in various environments (not just Java), PEM might be a better option.

wdyt?

Reply:
@sundy-li In general, we can support both trustStore and keyStore from JKS, as well as root certificates with client key and certificate.

Reply:
Not yet, I noticed that `clickhouse-jdbc` used  names: `sslmode ` etc.  So it's better to follow the same name to avoid too much ambiguoty.

Reply:
I understand your point. However, all other settings in our project use the snake_case style. Do you think that's a good approach?